Lib/fontTools/ttLib/tables/O_S_2f_2.py

```diff
@@ -25,6 +25,14 @@
 
 
 class Panose(object):
+    def __init__(self, panoseDict=None):
+        formatstring, names, fixes = sstruct.getformat(panoseFormat)
+        for name in names:
+            if panoseDict is not None and name in panoseDict:
+                setattr(self, name, safeEval(panoseDict[name]))
+            else:
+                setattr(self, name, 0)
```

Comment:
Was thinking more something like this:

```suggestion
    def __init__(self, **kwargs):
        _, names, _ = sstruct.getformat(panoseFormat)
        for name in names:
            setattr(self, name, kwargs.pop(name, 0))
        for k in kwargs:
            raise TypeError(f"Panose() got an unexpected keyword argument {k!r}")
```

Reply:
don't need to eval, we're not accepting strings in the init